index.html

```diff
@@ -18,11 +21,18 @@
   </div>
 </section>
 <div class="interstitial-image" style="background-image:url(/img/lcs-global-game-jam.jpg)"></div>
+<section id="broadband">
+  <div class="container">
+    {% include broadband.html %}
+  </div>
+</section>
+<div class="interstitial-image" style="background-image:url(/img/lcs-global-game-jam.jpg)"></div>
```

Comment:
Better question: why is this long page of text on the homepage.  Put something short here and link to broadband.html